plugins/motiontrail.lua

```diff
@@ -30,43 +30,69 @@ config.plugins.motiontrail = common.merge({
   }
 }, config.plugins.motiontrail)
 
+local cc_installed = pcall(require, 'plugins.custom_caret')
+local cc_conf = config.plugins.custom_caret
 
 local function lerp(a, b, t)
+  if not (a and b and t) then return 0 end
   return a + (b - a) * t
 end
```

Comment:
I think you could replace this with `common.lerp`.

Reply:
yup